scripts/remoteProcessPickerScript

```diff
@@ -1 +1,24 @@
-uname && if [ "$(uname)" = "Linux" ] ; then ps -axww -o pid=,flags=,comm=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa,args= ; exit; elif [ "$(uname)" = "Darwin" ] ; then ps -axww -o pid=,flags=,comm=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa,args= -c; fi
\ No newline at end of file
+#!/bin/sh
+uname && if [ "$(uname)" = "Linux" ]; then
+    ps -axww -o pid=,flags=,comm=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa,args= >/dev/null 2>/dev/null
+    ret=$?
+    if [ $ret -eq 0 ]; then
+        ps -axww -o pid=,flags=,comm=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa,args=
+    else
+
+        # Try to mimic the output of ps -axww -o pid=,flags=,comm=aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa,args= from /proc/$pid/ data
+        echo "        aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+        for pid in `cd /proc && ls -d [0-9]*`
+            do {
+                if [ -e /proc/$pid/stat ]
+                then
+                    flags=`awk '{print $9}' /proc/$pid/stat`;
+                    command=`awk '{print $2}' /proc/$pid/stat | tr -d '()'`
+                    args=`xargs -0 < /proc/$pid/cmdline | awk '{print $2}'`;
+                    printf "%5s %.1s %-49s %-27s \n\r" $pid $flags $command $args
```

Comment:
Could you add comments for what each of these lines is accomplishing. I am not an expert with linux utils and it would be helpful when referencing it in the future.

Reply:
Thanks for feedback, I have updated it with explanation of each line